### PR TITLE
fix dataset size to avoid dry runs

### PR DESCRIPTION
**Pull request by hadipash**

### CODE REVIEW

File: examples/opensora_hpcai/opensora/datasets/bucket.py [hunk 1]
```diff
@@ -128,4 +129,4 @@ def _bucket_split_function(video: np.ndarray) -> int:
         # video: F C H W
         return hashed_buckets[video.shape[0]][video.shape[2]][video.shape[3]]
 
-    return _bucket_split_function, list(range(1, cnt - 1)), batch_sizes
+    return _bucket_split_function, list(range(1, cnt - 1)), batch_sizes, median(batch_sizes)
```
Comment (SamitHuang) [inline examples/opensora_hpcai/opensora/datasets/bucket.py:132]: small probability:
- 10 HD video training samples
- median(batch_size) > 1, say 2, i.e. steps per epoch
- real-time sampled buckects, the corresponding batch size are all 1
- train 1 epoch
=>  only trained 5 steps and 5 vidoes are trained

Comment (hadipash) [reply examples/opensora_hpcai/opensora/datasets/bucket.py:132]: Yes, I am aware of it, but this is <ins>not a problem</ins>, as the current epoch step is used just for reference and doesn't affect the training flow in MindSpore, since the training continues as long as a dataloader returns data:
https://github.com/mindspore-ai/mindspore/blob/bd3c5dd1236bb5f2199b7f5ac2f2e6452879128f/mindspore/python/mindspore/train/model.py#L1100
The only thing it will affect is the printout log:
```shell
[2024-08-13 20:29:57] INFO: epoch 936 step 1, loss: 0.212903, loss scale: 1.
[2024-08-13 20:29:57] INFO: epoch 936 step 1, loss: 0.212903, loss scale: 1.
[2024-08-13 20:29:57] INFO: epoch 936 step 1, loss: 0.212903, loss scale: 1.
[2024-08-13 20:29:57] INFO: epoch 936 step 1, loss: 0.212903, loss scale: 1.
[2024-08-13 20:29:57] INFO: epoch 936 step 1, loss: 0.212903, loss scale: 1.

Comment (hadipash) [reply examples/opensora_hpcai/opensora/datasets/bucket.py:132]: So, this change is just to avoid a dry run and have a better estimation of the batch size than just a constant.

However, it would be much better to print out the epoch number and the global step instead of the current epoch step for coherence.
